### PR TITLE
adding FL cleaning functions

### PR DESCRIPTION
**Pull request by gilmourj**

### CODE REVIEW

File: app/api/ltc.py [hunk 1]
```diff
@@ -126,6 +126,33 @@ def collapse_rows_new_header_names(df_group, col_map):
...
+    else:
+        record['CTP_Facility_Type'] = None
+        record['Regulate'] = None
+    return record
+
+
+# clears any CMS IDs tied to facilities that are not nursing homes
+# this is optimized for FL - other states have different labels
+def clear_non_nh_cms_ids(record):
+    if ((record['State_Facility_Type'] != 'NH') and ~(record['CMS_ID'] == record['CMS_ID'])):
```
Comment (julia326) [inline app/api/ltc.py]: the 2nd half of that looks fishy - record['CMS_ID'] would always equal itself?

File: app/api/ltc.py [hunk 2]
```diff
@@ -241,7 +274,7 @@ def do_aggregate_outbreaks(df):
 
     # TODO: if more than FL needs special treatment before aggregating outbreaks, factor this out
     # into something nicer
-    state = set(df['State']).pop()
+    state = {x for x in set(df['State']) if x==x}.pop()
```
Comment (julia326) [inline app/api/ltc.py]: sorry I got confused by this, what was the context where this came up?

File: app/api/ltc.py [hunk 3]
```diff
@@ -126,6 +126,33 @@ def collapse_rows_new_header_names(df_group, col_map):
     return new_df_subset
 
 
+# cleans up CTP Facility Types and federal/state regulated
+# this is optimized for FL - other states have different labels
+def state_to_ctp(record):
```
Comment (julia326) [inline app/api/ltc.py]: maybe we can rename to `FL_state_to_ctp` since it's a FL specific function?

**Review by julia326 — COMMENTED** [review]
Thank you @gilmourj ! This looks good, just a couple of questions/comments

File: app/api/ltc.py (hunk 1 above)
Comment (gilmourj) [reply app/api/ltc.py]: I was using the fact that nan != nan but I don't actually need that check at all. ty!

File: app/api/ltc.py (hunk 2 above)
Comment (gilmourj) [reply app/api/ltc.py]: when I ran this on the FL Facility sheet, `set(df['State']).pop()` was returning nan so I threw this in to only include non-nan elements in the set and then pop

File: app/api/ltc.py (hunk 3 above)
Comment (gilmourj) [reply app/api/ltc.py]: good idea!

**Review by julia326 — APPROVED** [review]

**Pull request merged**